packages/google_maps_flutter/google_maps_flutter_ios/ios/google_maps_flutter_ios.podspec

```diff
@@ -36,6 +36,7 @@ Downloaded by pub (not CocoaPods).
   s.xcconfig = {
     'LIBRARY_SEARCH_PATHS' => '$(inherited) $(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)/ $(SDKROOT)/usr/lib/swift',
     'LD_RUNPATH_SEARCH_PATHS' => '$(inherited) /usr/lib/swift',
+    'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) FGM_USING_COCOAPODS=1'
```

Comment:
Note this `FGM_USING_COCOAPODS` preprocessor definition used to detect if cocoapods are used or not.
This is used for example at: https://github.com/flutter/packages/pull/8288/files#diff-dcc8100c7e606a3918ac1f3f2cd0aa16c2c6e551e09b9ff717f724a671ce5323R8